avocado/core/safeloader.py

```diff
@@ -118,9 +118,9 @@ def add_imported_object(self, statement):
             for _ in range(level - 1):
                 path = os.path.dirname(path)
         for name in statement.names:
-            path = os.path.join(path, name.name.replace('.', os.path.sep))
+            full_path = os.path.join(path, name.name.replace('.', os.path.sep))
             final_name = self._get_name_from_alias_statement(name)
-            self.imported_objects[final_name] = path
+            self.imported_objects[final_name] = full_path
```

Comment:
Ouch!!! Nice catch! I was trying to understand, but I got it. `path` was already defined on the parent scope! :+1:

Reply:
Same here :)